app/models/site_setting.rb

```diff
@@ -133,6 +133,7 @@ def self.enable_s3_uploads
     end
 
     def self.absolute_base_url
+      return nil if !SiteSetting.Upload.enable_s3_uploads
```

Comment:
Can we add a test case here because it isn't clear to me what we're trying to fix here.

Reply:
Actually I figured out it's not really needed so I'll just remove it.